packages/formatter-json/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "sonarwhal formatter that prints the JSON object of the results",
+  "dependencies": {
+    "lodash": "^4.17.5"
```

Comment:
Can we change this to using only the method(s) that we are using instead of the whole lodash?